internal/cli/setup/kustomize/setup.go

```diff
@@ -0,0 +1,22 @@
+package setup
+
+import (
+	"github.com/kyma-project/cli/internal/kustomize"
+	"github.com/kyma-project/cli/pkg/step"
+)
+
+const (
+	DefaultNewStepMsg     = "Setting up kustomize..."
+	DefaultStepSuccessMsg = "Kustomize ready"
+)
+
+// ExtendCmd is used to extend the CLI command with the SetupKustomize() function
```

Comment:
IMO this `ExtendCmd` struct is over engineered, you can simply define a `func SetupKustomize(s step.Step) error`

Reply:
As described above, it's my take on introducing some shared logic into existing command structure.
Essentially a way to easily add it to the commands just by extending their types.
A single "mixin" doesn't bring that much value, but if we have more of them, then you can tell what the command does, at least partially, just by looking at it's type.
I'd prefer to do it with a proper mixin mechanism known from other languages, but golang doesn't have that.
Let me know what you think :)

Reply:
https://github.com/kyma-project/cli/pull/1473/files#r1044166902